spec/controllers/collections_controller_spec.rb

```diff
@@ -94,7 +97,10 @@ def to_solr(solr_doc={})
       @asset1 = GenericFile.create!
       @asset2 = GenericFile.create!
       post :create, batch_document_ids: [@asset1,@asset2], collection: {title: "My Secong Collection ", description: "The Description\r\n\r\nand more"}
-      expect(assigns[:collection].members).to eq([@asset1,@asset2])
```

Comment:
I'm inclined to think order matters - or at least we probably don't want things we post to end up in the model in opposite order. Does the ordered reader return the same thing?